src/views/AuthorisationRedirect.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div>
+    <h1>
+      {{Status}}
+    </h1>
+  </div>
+</template>
+
+<script lang="ts">
+
+import {Options ,Vue} from "vue-class-component";
+
+@Options({
+  components: {
+    AuthorisationRedirect
+
+  },
+  props:{
+    github: {
+      type: String,
+      required: true,
+      default: null
+    }
+  },
+  Status: ""
+
+})
+
+export default class AuthorisationRedirect extends Vue{
```

Comment:
For now, this is good, In the future please use a service class in a separate file